library/build.gradle

```diff
@@ -23,10 +23,10 @@ repositories {
 
 dependencies {
     api "org.apache.avro:avro:1.11.3"
-    implementation 'ch.qos.logback:logback-classic:1.2.9'
-    testImplementation 'org.json:json:20230618'
+    implementation 'ch.qos.logback:logback-classic:1.3.14'
```

Comment:
see https://github.com/DataBiosphere/java-pfb/pull/19#pullrequestreview-1779802395 for why this is `1.3.14` and not `1.4.14`